sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -57,44 +84,114 @@ export default {
     const resetIdleCheckerTimer = () => {
       clearInterval(interval.value);
       if (isAuthenticated.value) {
-        /* eslint-disable */
-        interval.value = setInterval(checkIdle, 30000);
-        /* eslint-enable */
+        interval.value = setInterval(checkIdle, 1000);
       }
     };
 
+    watch(
+      () => modalOpen.value,
```

Comment:
mm, what exactly is this value? `modalOpen`,

Reply:
The value is true if the modal is open for the user.